### ISSUE TITLE
[Accessibility] A11y_.Net Editor_Screen Reader: When Narrator/NVDA is on user needs to press ESC key two times to close command palette

### ISSUE THREAD

**Issue by Tanushagottipati** [dotnet/try#120]
**Environment Details**
https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
**Browser:** Edge (41.16299.402.0)
Chrome (67.0.3396.87)
**Screen Reader:** Narrator
NVDA

**Additional Details:**
MAS Violated: MAS1.3.2, MAS2.1.1

**Repro steps:**
1.	Hit the URL https://docs.microsoft.com/en-us/dotnet/csharp/quick-starts/hello-world?tutorial-step=4User
2.	Tab till “Enter focus mode” and press enter
3.	Type any code and press F1 and on the Screen reader
4.	Check whether user is able to close the Collaboration list on pressing Esc only once when screen reader is on

**Actual Result:-**
When screen reader is on user needs to press ESC key two time to close command palette

**Expected Result:-**
User should be able to close the list items on pressing the dropdown only once. Command to perform any operation should be changed when Screen reader is on.

**User Impact:-**
Screen Reader users find difficulty in using application if user needs to operate controls twice to perform an action.

**MAS Reference:-**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={1ee227a7-e3c5-47b8-ab18-48ca8595077b}

https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

[When screen reader is on user needs to press ESC key two time to close command palette.pptx](https://github.com/dotnet/try/files/2191424/When.screen.reader.is.on.user.needs.to.press.ESC.key.two.time.to.close.command.palette.pptx)

**Comment by LadyNaggaga** [comment]
**Update**
Issue has been flagged and assigned to the Monaco repo.
https://github.com/Microsoft/monaco-editor/issues/1167

**Comment by LadyNaggaga** [comment]
**Update**
Issue has been assigned on the Monaco repo
https://github.com/Microsoft/monaco-editor/issues/1167

**Comment by LadyNaggaga** [comment]
1.	Importance of experience (High or Low Impact). 
Impact: Low we haven't received any customer complaints directly to the service. The .NET online editor is the service behind editor and we do not own or control the UI experience. 
Describe scenario: Please the description above
2.	Quality of Experience (High or Low Impact). 
3.	[Link to AAT ](https://cela-ra-aat-prod.azurewebsites.net/#/workflow/93711e75-9cb7-4061-b1e9-53177a087cae)
4.	Any alternate solutions or technologies evaluated
Not applicable 
5.	Ensure all evidence for hand-off to external team is provided in the bug
Issue has been flagged and assigned to the Monaco repo.
https://github.com/Microsoft/monaco-editor/issues/1167

**Comment by umasarath52** [comment]
This was discussed with Ryan and approved to close as this is non blocking scenario.
